player/public/js/ekstepRendererApi.js

```diff
@@ -1056,6 +1056,9 @@ window.EkstepRendererAPI = {
 			}
 			EkstepRendererAPI.hideEndPage()
 			content = contentObj
+			if (content.mimeType === "application/vnd.ekstep.ecml-archive") {
```

Comment:
presently canvas show/hide is causing an issue, we are again doing the same by adding one more event. Code is already messed, don't mess even further.